llama-demo-apps/Llama2_Gradio.ipynb

```diff
@@ -0,0 +1,96 @@
+{
```

Comment:
@jeffxtang couple of points: a- wonder if we could have used gradio in the other notebooks where we used replicate or query you.com b - if we need to keep it I believe we would need some more context on gradio/ its chatpipeline.

Reply:
I am suppose to bring up a PR for a more comprehensive chatbot example using Gradio and search API. Shall we merge some of the works here based on your suggestion?

Reply:
I agree that if we want to have gradio example, it might help to provide more comprehensive example or add some more explanations to the existing one.

Reply:
Chester will create a PR soon with a full-fledged gradio-based llama app. so here i just intended it to the simplest self-sufficient teaser demo. using gradio in the other notebooks will certainly be nice but maybe in a future update.